### PR TITLE
Main layout complete

### PR DESCRIPTION
**Pull request by JVLoe**
besides adding links to buttons and some more aesthetic features - mainly finished

### CODE REVIEW

**Review by SlothSan — CHANGES_REQUESTED** [review]
Please setup .GITIIgnore (See Rayna's notes on how to do this and ignore .DS_Store

**Review by SlothSan — CHANGES_REQUESTED** [review]
Please create a img directory in your portfolio and move all your images to here and change your link to use that instead.

**Review by SlothSan — CHANGES_REQUESTED** [review]
Great work, looks good. 

Please see all my proposed changes, and shout me if they don't make sense to you.

Cheers, 

Mike.

File: Portfolio.css [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+a:hover, a:active {
+  color: rgba(15, 233, 228, 1);
+}
+
+
+nav {
+    position: fixed;
+    background-color:rgba(170, 0, 255, 0.4);
+    padding: 10px;
+	width: 100%;
```
[the remark this answers is not in the document]
Comment (SlothSan) [reply Portfolio.css:27]: Please ignore, it is just Github.

File: Portfolio.css [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+	padding: 20px;
+	gap: 10px;
+	font-size: 1.3rem;
+	margin-top: 10px;
+}
+
+
+.top_contact_buttons {
+	display: flex;
+	flex-wrap: nowrap;
```
[the remark this answers is not in the document]
Comment (JVLoe) [reply Portfolio.css:46]: nope, its nowrap

File: Portfolio.html [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+			<h2 class="intro2">Here you can explore all my projects</h2>
+			<h3 class="intro3">Please enjoy   <img src="smiley-face.png" alt="Smiley Face" class="smiley-face"></h3>
+		</div>
+	</header>
+	<main>
+		<section id="about">
+			<div class="about_container">
+				<h2 class="about_header">About</h2>
+				<div class="about_words">
+					<p>Hi, i'm Jo. I'm a student at iO Academy, training to become a web developer. Throughout this course, I will 				learn HTML, CSS, JavaScript, and PHP. I will demonstrate my projects below.</p>
```
[the remark this answers is not in the document]
Comment (JVLoe) [reply Portfolio.html]: wait, which part do you mean?

File: Portfolio.html [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+			</div>
+		</section>
+		<section id="portfolio" class="portfolio_top">
+			<div>
+				<h2 class="portfolio_header">Portfolio</h2>
+				<p>Here is a collection of projects i've worked on</p>
+			</div>
+		</section>
+		<section class="portfolio_projects">
+			<div class="project1">
```
[the remark this answers is not in the document]
Comment (JVLoe) [reply Portfolio.html:62]: changed all 4 to be "project"

**Review by SlothSan — APPROVED** [review]
Approved - thanks for making all the changes.

**Review by SlothSan — APPROVED** [review]
Approved - changes have been made - good job.

**Review by SlothSan — APPROVED** [review]
Approved

**Review by SlothSan — APPROVED** [review]
Approved - all changes are in !

**Review by norayna — APPROVED** [review]
approving without reading to hopefully make it go through

**Pull request merged**